thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -1382,17 +1382,18 @@ Sizzle is good for finding elements for a selector, but not so good for telling
 		$(document).triggerHandler(event);
 	});
 
-})(jQuery);;
+})(jQuery);
 
 
 /* src/domevents/jquery.entwine.domevents.maybechanged.js */
 
 (function($){
 
-	/** What to call to run a function 'soon'. Normally setTimeout, but for syncronous mode we override so soon === now */
-	var runSoon = window.setTimeout;
+  // Helper function to run a function "soon". This is effectively using setTimeout, but ensuring it doesn't run
+  // before DOMContentLoaded (via jQuery.ready())
+  var runSoon = (fn, delay) => $(_ => window.setTimeout(fn, delay));
```

Comment:
Keep the note about the fact that it's overridden for syncronous mode.

Reply:
Probably also worth adding a link to https://api.jquery.com/jQuery/#jQuery3 - I had to go hunting to find out why this actually works, as I wasn't familiar with this alternative to `$(document).ready()`

Reply:
I updated the comment to mention `$(document).ready()` but forgot the link 😅 hopefully the comment is okay now anyway, links sometimes go stale anyway